### DISCUSSION TITLE
Create new repo to work together on an OSPO Mind Map

### DISCUSSION THREAD

**Discussion by anajsana** [todogroup/ospology#75] [Community Proposals]
Hi everyone!

Yesterday I shared on [SM](https://www.linkedin.com/posts/ana-jim%C3%A9nez-santamar%C3%ADa_ospo-mind-map-activity-6912789794024501248-et50) an OSPO mind map idea I created based on some of the TODO resources available and previous experience 🙂 

[OSPO Mind Map.pdf](https://github.com/todogroup/ospology/files/8348739/OSPO.Mind.Map.pdf)

Some people from the open source community would love to work together and  expand and improve this mind map.

## Proposal

Create a new repo under TODO Group GH Org to allow people to contribute to this, and define a process and roadmap.

## Final Resolution (2022-03-29)

Created a [new folder](https://github.com/todogroup/ospology/tree/main/ospo-mindmap) under the 'ospology' repo to contribute to the mind map

**Comment by winterrocks** [comment]
There are 2-3 mind map filetypes that I am aware of. I am not sure which one is the most popular, but it would be good to have one that has some good free (even open source) mind map tools available. 

I have used XMind application in the past, but I have no mind mapping religion.

**Reply by anajsana** [reply]
Good point Jari! I'm familiar with Xmind and IMO, it works fine for this purpose. I think other options such as compedium or freeplane are open source AFAIK and tools to consider 🙂

**Comment by awright** [comment]
> Hi everyone!
> 
> Yesterday I shared on SM an OSPO mind map idea I created based on some of the TODO resources available and previous experience 🙂
> 
> [OSPO Mind Map.pdf](https://github.com/todogroup/ospology/files/8348739/OSPO.Mind.Map.pdf)
> 
> Some people from the open source community would love to work together and expand and improve this mind map.
> 
> ##Proposal
> 
> Create a new repo under TODO Group GH Org to allow people to contribute to this, and define a process and roadmap.

@anajsana I really like this mind map. I'd be curious to talk more about your plans here during our touchpoint today.

**Reply by anajsana** [reply]
@awright we can add it to the agenda and share the outcomes from that discussion here 🙂but I don't know if we would have enough time today by looking at the agenda items 🤔

**Reply by anajsana** [reply]
Are there any items you'd like to see on the mind map? maybe we can discuss it here?

**Comment by jsmanrique** [comment]
I really like it, and I'd love to explore expanding it somehow.

Wouldn't it be great if the mindmap could be managed as "code" is? To achieve it, I think it might be possible to have it on a "text" format that could be used to generate the graphic view.

I've been originally thinking about [mermaid](https://mermaid-js.github.io/mermaid/#/), supported both by GitLab and GitHub. But, AFAIK, it doesn't support mindmaps.

Other options could be [text2mm](https://text2mm.com/) or similar. Any ideas?

**Reply by anajsana** [reply]
Thanks for sharing these resources and offering your help! I like a lot the idea of using [text2mm](https://text2mm.com/)! I've seen that It can be downloaded into `.txt`, which can make easier for people to make contributions to the content via PRs 👍 

Also, it works with markdown syntax, and branches are configured using a simple indentation system: https://text2mm.com/help

**Comment by geekygirldawn** [comment]
I love the idea of people collaborating on a mind map, but this feels like an activity that could be a directory in an existing repo, rather than a new repo. We just archived a bunch of repos that were used for a single activity and later abandoned, so I'd like to avoid creating new repos unless we really need one. We can always break it out into a new repo later if it gains a lot of traction and becomes bigger.

I'm thinking one of these repos might be a good fit: OSPOlogy or opensource-research?

**Reply by anajsana** [reply]
Good point @geekygirldawn (and thanks a lot for reviewing this proposal!) Based on your suggestions, I like the idea to choose `ospology` since it is intended to be the starting point for the "study of OSPOs", while `opensource-research` contains more "a list of research around open source and relevant quotes".

**Reply by justaugustus** [reply]
Agreed with @geekygirldawn and I see this is resolved with https://github.com/todogroup/ospology/discussions/75#discussioncomment-2461834.

**Comment by caniszczyk** [comment]
+1

On Tue, Mar 29, 2022 at 11:44 AM Ana Jimenez Santamaria <
***@***.***> wrote:

> Good point @geekygirldawn <https://github.com/geekygirldawn> (and thanks
> a lot for reviewing this proposal!) Based on your suggestions, I like the
> idea to choose ospology since it is intended to be the starting point for
> the "study of OSPOs", while opensource-research contains more "a list of
> research around open source and relevant quotes".
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/todogroup/ospology/discussions/75#discussioncomment-2459980>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAPSIJNH6GV6E4J7MGRT5DVCLGHVANCNFSM5RTPGEWQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Cheers,

Chris Aniszczyk
https://aniszczyk.org

**Comment by anajsana** [comment]
FYI: Created a [new folder](https://github.com/todogroup/ospology/tree/main/ospo-mindmap) under the 'ospology' repo to contribute to the mind map 👍
